test/quantization/fx/test_quantize_fx.py

```diff
@@ -5576,6 +5576,28 @@ def test_elu(self):
     def test_leaky_relu(self):
         self._test_activation_impl(nn.LeakyReLU, F.leaky_relu, nnq.LeakyReLU, torch.ops.quantized.leaky_relu)
 
+    def test_prelu(self):
```

Comment:
can we reuse `_test_activation_impl` here?

Reply:
Thanks for the suggestion. PReLU needs additional arguments since it has weight. For example, `num_parameters` for module constructor and `weight` for `forward` of functional op. How do you suggest adding them? By adding `**kwargs` or `*args` to `_test_activation_impl`? Then we need to do some `if ... else ...` to accommodate PReLU. Do we have to consider other activations with additional parameters?